package.json

```
@@ -2,46 +2,77 @@
   "name": "@warp-ds/icons",
   "type": "module",
   "version": "0.0.1",
+  "repository": "git@github.com:warp-ds/icons.git",
   "description": "WARP's icon set",
   "main": "index.js",
   "files": [
-    "dist"
+    "dist",
+    "vue",
+    "react",
+    "elements"
   ],
   "exports": {
-    ".": "./index.js"
+    ".": "./index.js",
+    "./react": "./react/index.js",
+    "./vue": "./vue/index.js",
+    "./elements": "./elements/index.js",
+    "./elements/*": "./elements/*.js",
+    "./package.json": "./package.json"
   },
   "scripts": {
     "import": "node ./scripts/import.js",
     "build": "node ./scripts/build.js",
-    "render": "node preview/render.js",
     "dev": "vite",
     "vite:build": "vite build",
-    "preview": "pnpm run build && vite build && vite preview"
+    "preview": "pnpm run build && vite build && vite preview",
```

Just to align with other examples :) 
```suggestion
    "preview": "pnpm build && vite build && vite preview",
```

---

Fixed in [da0b941](https://github.com/warp-ds/icons/pull/4/commits/da0b941617db26a10159664a6b7a588aa76c23bf). Here I also renamed the command that shows a preview of all icons to `pnpm dev`. This way all our repos have a similar command to see the output of that package locally. Tested that the output is the same when using `pnpm build && vite` as in `pnpm build && vite build && vite preview`.